templates/_scoreboard.erb

```diff
@@ -22,7 +23,12 @@
           <% end %>
         </tr>
       <% end %>
-
     </table>
   <% end %>
 </div>
+
+<script>
+    $(document).ready(function() {
+        $('.table').DataTable({bFilter: false, bInfo: false, bPaginate: false});
```

Comment:
@ometa The values here can be changed to enable Searching, Pagination and to display some information about the number of rows. I have set all to false for now as I think pagination and info are unnecessary but bFilter brings in a search box that could be useful. This also applies to the table the users see in the all_checkins.erb file.